src/OmniSharp.Http/OmniSharp.Http.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net46;netstandard1.6</TargetFrameworks>
+        <WarningsAsErrors>true</WarningsAsErrors>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <!-- Needed for Microsoft.Composition -->
+        <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.6' ">$(PackageTargetFallback);dotnet5.4;portable-net45+win8</PackageTargetFallback>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="1.1.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="1.1.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="1.1.0" />
+        <ProjectReference Include="..\OmniSharp.Host\OmniSharp.Host.csproj" />
```

Comment:
My initial thinking is to have the hosting Api's here.  Which will allow us to have different startup projects.  Which will then be packaged with their specific dependencies.

* `OmniSharp.Http` - Starts an http based server (kestrel)
* `OmniSharp.Stdio` / `OmniSharp` (for legacy reasons) - Starts an stdio based server
* `OmniSharp.Lsp`* - Starts an LSP based server (could be integrated into `Stdio` potentially.
* `OmniSharp.Sockets`* - Starts up a server that runs over a traditional socket
* `OmniSharp.WebSockets`* - Starts up a server that runs over a WebSocket.

Now these are cool endgame ideas, which may or may not require large additional assemblies / libraries, ie Kestrel.  The three big things we need to support are Http, Stdio, and in the future Language Server Protocol.